boxyhq/jackson
# feat(docker-compose): run jackson service

## What does this PR do?

This PR intends to facilitate the deployment of the Jackson service, using docker compose. I transferred the docker command provided in the documentation into a generic docker compose file. I'm not sure what makes the most sense regarding the DB settings, so for now I have only added the Postgres env variables.

Fixes #61 (issue)

## Type of change


## How should this be tested?


1. Start your postgres DB
1. In your root folder, run:
```shell
docker compose up
```
This should create a container named `jackson`, running on port `5225`.
3. The service is accessible at the following address: http://localhost:5225/. And if you go to `http://localhost:5225/api/health`, it will return your current version of Jackson (for me it was `{version: "1.13.0"}`).

## Checklist:
